### DISCUSSION TITLE
Is it possible to mark the channel as "unread" without sending a message?

### DISCUSSION THREAD

**Discussion by frontez** [discord-net/Discord.Net#2243] [Q&A] [answered]
Do you  guys know if it is possible to mark the channel as "unread" (like when there is a new message in the channel)? by discord api, but not sending an actual message

Because for my PUG bot I dont send new messages, I only update the existing message when players joining/leaving the queue so the channel doesnt change it's state to "unread" and people dont see that someone queued the pug from outside.

![image](https://user-images.githubusercontent.com/15384653/163144405-a8e32f99-952d-488d-bc74-d59652bac62e.png)

**Comment by exsersewo** [comment, marked as answer]
Channels being read or not is a client side thing, you will have to send a message and delete it to mark it as unread.
